packages/react-scripts/config/paths.js

```diff
@@ -78,9 +78,12 @@ module.exports = {
   dotenv: resolveApp('.env'),
   appPath: resolveApp('.'),
   appBuild: resolveApp('build'),
+  devAppBuild: resolveApp('dev'),
```

Comment:
This is to load the development version of the extension. See further in this PR for more details